docs/index.html

```diff
@@ -857,7 +890,7 @@ <h4>Comments</h4>
 <article class="relation">
   <h3 id="meronym"><a href="#meronym">Meronym</a> (<span class="rel-id">meronym</span>)<a class="toplink" href="#">back to top</a></h3>
   <div class="description">
-    <span class="short definition">"Y makes up a part of X"</span>
+    <span class="short definition">"A word that denotes a constituent part or a member of something."</span>
```

Comment:
These definitions and examples are definitely an improvement over the existing text, but they belong in the data files (e.g., `gwadoc/doc_en.py`) and not in this autogenerated `index.html` file.